src/Peering/Peering/Exchange/SetAzureExchangePeeringConnectionCommand.cs

```diff
@@ -171,16 +171,13 @@ private PSExchangeConnection UpdateMD5Authentication()
         /// </returns>
         private PSExchangeConnection UpdateIpV4Prefix()
         {
-            if (this.InputObject is PSExchangeConnection inputObject)
+            if (this.InputObject.GetType().Equals(typeof(PSExchangeConnection)))
```

Comment:
Why use direct type comparsion instead of `is`? Except for extremely rare cases, `is` is what you want.